velox/type/Timestamp.h

```diff
@@ -148,7 +148,8 @@ struct Timestamp {
   };
 
   std::string toString(
-      const Precision& precision = Precision::kNanoseconds) const {
+      const Precision& precision = Precision::kNanoseconds,
+      int timeWithTimezoneFormatFlag = 0) const {
```

Comment:
This can be bool variable to make it cleaner.

Maybe call the variable printTimezone.

Reply:
Done